src/TwilioVideo.android.js

```diff
@@ -204,6 +213,10 @@ class CustomTwilioVideoView extends Component {
     this.runCommand(nativeEvents.sendString, [message]);
   }
 
+  sendEncodedString(message) {
+    this.runCommand(nativeEvents.sendEncodedString, [message]);
```

Comment:
Sending will be the reverse of receiving: take a `UInt8Array` as input, base64-encode it, and send it to native code. Using the `base64-js` library I mentioned earlier, this would be:

```js
  sendBinary(message) {
    this.runCommand(nativeEvents.sendBinary, [base64js.fromByteArray(message)]);
  }
```

And of course rename all the instances of `sendEncodedString` to `sendBinary` everywhere.

Reply:
Took out sendEncodedString since I realized the ticket only specifies receiving data, but I will keep this in mind when we tackle it.